apps/src/lib/kits/maker/toolkit.js

```diff
@@ -164,34 +164,29 @@ function confirmSupportedBrowser() {
  * @returns {Promise.<MakerBoard>}
  */
 function getBoard() {
+  const boardConstructor =
+    project.getMakerAPIs === MB_API ? MicroBitBoard : CircuitPlaygroundBoard;
+
```

Comment:
Now that Micro:Bit also includes the WebSerial pathway, refactor this function to include this addition.